allennlp/training/trainer.py

```diff
@@ -175,89 +175,89 @@ class GradientDescentTrainer(Trainer):
     data_loader : `DataLoader`, required.
         A pytorch `DataLoader` containing your `Dataset`, yielding padded indexed batches.
 
```

Comment:
I added all of these blank lines in the docs at one point because it was necessary to get things to render properly. Does that make a difference anymore? It'd be nice if we could remove the need for them.

Reply:
Those aren't necessary anymore.